packages/devtools_app/lib/src/shared/inspector_preferences_controller.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/foundation.dart';
+
+import '../../devtools_app.dart';
+
+class InspectorPreferencesController {
```

Comment:
feel free to put this class directly into preferences.dart. It is common practice in dart to have more than once class in a file. This will be a natural fit if we make InspectorPreferencesController a class member of Preferences.